src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int sizeOfList;
+    private Node firstNode;
+    private Node lastNode;
```

Comment:
```suggestion
    private Node<T> firstNode;
    private Node<T> lastNode;
```
don't forget ur type parameter